### ISSUE TITLE
evaluation metrics

### ISSUE THREAD

**Issue by wangzhen699** [QtacierP/LED#1]
Thanks for your work,could you share the code about evaluation metrics,such as FIQA ，OCSD，VSD， FCNR and DRA.

**Comment by QtacierP** [comment]
We will be updating certain non-parameter metrics such as FCNR. However, as all other metrics require a heavily pre-trained model and task-specific metrics, it would be difficult to merge them into a single repository. Nevertheless, we recommend using the following repositories to calculate the Dice/Acc on the test set: [IterNet](https://github.com/conscienceli/IterNet) for VSD, [MNet](https://github.com/HzFu/MNet_DeepCDR) for OCSD, [MCFNet](https://github.com/HzFu/EyeQ) for FIQA, and [Lesion CL](https://github.com/YijinHuang/Lesion-based-Contrastive-Learning) for DRA. All these repositories have provided pre-trained models and evaluation codes for your convenience.

**Comment by QtacierP** [comment]
Of course, you can train your own evaluation model. We found that a model trained on high resolution will give better performance for quality assessment. Experimentally, all evaluation models we used were trained on approximately 512x512.
